tools/pubsys/src/aws/publish_ami/mod.rs

```diff
@@ -23,52 +24,49 @@ use std::collections::{HashMap, HashSet};
 use std::fs::File;
 use std::iter::FromIterator;
 use std::path::PathBuf;
-use structopt::{clap, StructOpt};
 
-#[derive(Debug, StructOpt)]
+#[derive(Debug, Parser)]
 pub(crate) struct ModifyOptions {
     /// User IDs to give/remove access
-    #[structopt(long, use_delimiter = true, group = "who")]
+    #[arg(long, value_delimiter = ',', group = "who")]
     pub(crate) user_ids: Vec<String>,
     /// Group names to give/remove access
-    #[structopt(long, use_delimiter = true, group = "who")]
+    #[arg(long, value_delimiter = ',', group = "who")]
     pub(crate) group_names: Vec<String>,
     /// Organization arns to give/remove access
-    #[structopt(long, use_delimiter = true, group = "who")]
+    #[arg(long, value_delimiter = ',', group = "who")]
     pub(crate) organization_arns: Vec<String>,
     /// Organizational unit arns to give/remove access
-    #[structopt(long, use_delimiter = true, group = "who")]
+    #[arg(long, value_delimiter = ',', group = "who")]
     pub(crate) organizational_unit_arns: Vec<String>,
 }
 
 /// Grants or revokes permissions to Bottlerocket AMIs
-#[derive(Debug, StructOpt)]
-#[structopt(setting = clap::AppSettings::DeriveDisplayOrder)]
-#[structopt(group = clap::ArgGroup::with_name("mode").required(true).multiple(false))]
-#[structopt(group = clap::ArgGroup::with_name("who").required(true).multiple(true))]
-pub(crate) struct PublishArgs {
+#[derive(Debug, ClapArgs)]
+#[group(required = true, multiple = true)]
+pub(crate) struct Who {
     /// Path to the JSON file containing regional AMI IDs to modify
-    #[structopt(long)]
+    #[arg(long)]
     ami_input: PathBuf,
 
     /// Comma-separated list of regions to publish in, overriding Infra.toml; given regions must be
     /// in the --ami-input file
-    #[structopt(long, use_delimiter = true)]
+    #[arg(long, value_delimiter = ',')]
     regions: Vec<String>,
 
     /// Grant access to the given users/groups
-    #[structopt(long, group = "mode")]
+    #[arg(long, group = "mode")]
     grant: bool,
     /// Revoke access from the given users/groups
-    #[structopt(long, group = "mode")]
+    #[arg(long, group = "mode")]
     revoke: bool,
 
-    #[structopt(flatten)]
+    #[command(flatten)]
```

Comment:
Is `command` used here to get access to `flatten`?

Reply:
Yes, `clap` uses more specific attribute names to differentiate some of the options.